cmd/agent/app/integrations.go

```diff
@@ -115,6 +121,59 @@ var showCmd = &cobra.Command{
 	RunE:  show,
 }
 
+func detectEnvironment() error {
+	rootDir, _ = executable.Folder()
+	for {
```

Comment:
is the place for `reqAgentReleaseFile` dynamic or different betweent windows/linux ? This `for` will got back to the root filesystem.

Reply:
It is dynamic relative to the Agent binary, yes

Reply:
If it's dynamic on windows don't we know exactly where it is relative to the binary ?

Reply:
This way makes it so we don't have to hard code a different location for every platform.